### PR TITLE
Document error for agent config file 's config description

### PR DESCRIPTION
**Pull request by seifeHu**
### Bug fix
- Bug description.
https://github.com/apache/skywalking/issues/3942

In
https://github.com/apache/skywalking/blob/v6.5.0/docs/en/setup/service-agent/java-agent/README.md

logging.dir | Log files directory. Default is blank string, means, use "system.out" to output logs. | ""
-- | -- | --

### CODE REVIEW

File: docs/en/setup/service-agent/java-agent/README.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ property key | Description | Default |
 `logging.level`|The log level. Default is debug.|`DEBUG`|
 `logging.file_name`|Log file name.|`skywalking-api.log`|
 `logging.output`| Log output. Default is FILE. Use CONSOLE means output to stdout. |`FILE`|
-`logging.dir`|Log files directory. Default is blank string, means, use "system.out" to output logs.|`""`|
+`logging.dir`|Log files directory. Default is blank string, means, use "{theSkywalkingAgentJarDir}/logs  " to output logs. {theSkywalkingAgentJarDir} is the directory where the skywalking agent jar file is located |`""`|
```
Comment (seifeHu) [inline docs/en/setup/service-agent/java-agent/README.md]: the description is an Error .In fact, the log will be output to the  "agentJarDir/logs "

**Comment by wu-sheng** [comment]
The class comment is wrong too, org.apache.skywalking.apm.agent.core.conf.Config#Logging#DIR. Could you fix this too?

**Comment by seifeHu** [comment]
> The class comment is wrong too, org.apache.skywalking.apm.agent.core.conf.Config#Logging#DIR. Could you fix this too?

ok . I pull an other one pull request

**Comment by wu-sheng** [comment]
This one is good, you don't need a new one.

**Pull request merged**